contracts/clearing/BrokerVault.sol

```diff
@@ -83,13 +98,23 @@ contract BrokerVault is IVault2, ISettle, Ownable2Step, ReentrancyGuard {
 		INitroTypes.RecoveredVariablePart calldata candidate
 	) external {
 		uint256 channelId = NitroUtils.getChannelId(fixedPart);
+		if (performedSettlements[channelId]) {
+			revert SettlementAlreadyPerformed(channelId);
+		}
+
+		if (fixedPart.participants[1] != broker) {
+			revert BrokerNotParticipant(fixedPart.participants[1], broker);
+		}
```

Comment:
You can simplify both of these checks by following the `require(statementToBeTrue, CustomError())` syntax

Reply:
cb9be30